### DISCUSSION TITLE
not able to send trasactions using signer

### DISCUSSION THREAD

**Discussion by anand008186** [WalletConnect/walletconnect-monorepo#3977] [Developer Support]
```javascript
import { useWeb3ModalSigner } from '@web3modal/ethers5/react';
 const {signer} = useWeb3ModalSigner()
 const contract = new ethers.Contract( contract_Address,abi,signer);
 const receipt = await contract.multiSwap(
    [...sellTokenAddress],
    [...buyTokenAddress],
    [...allowanceTarget],
    [...to],
    [...data],
    [...sellAmount],
    sendObj
  );
  await receipt.wait();
```
I am trying to send this transaction using web3Modal signer hooks but getting error (webpack-internal:///(app-pages-browser)/./node_modules/@ethersproject/providers/lib.esm/base-provider.js:28:58)

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/77172217/557745af-aea9-4049-b525-9df1121f1def)

**Comment by glitch-txs** [comment]
tx works find for me https://github.com/glitch-txs/web3modal-ethers/blob/main/src/Transaction.tsx
probably an issue on ethers or it's implementation, maybe try opening a discussion in ethers repository or their discord
https://github.com/ethers-io/ethers.js
